### PR TITLE
Make glob requirements clearer

### PR DESCRIPTION
**Pull request by niceking**
### Description

As raised by @TJC in https://github.com/buildkite/test-splitter/issues/52, the phrasing of what we expect the glob to be could be clearer

### CODE REVIEW

**Review by wooly — APPROVED** [review]

**Review by amybiyuliu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ The available Go binaries
...
-| `BUILDKITE_SPLITTER_TEST_FILE_PATTERN` | `spec/**/*_spec.rb` | Optional, glob pattern for discovering test files that need to be executed. </br> *It accepts pattern syntax supported by [zzglob](https://github.com/DrJosh9000/zzglob?tab=readme-ov-file#pattern-syntax) library*. |
-| `BUILDKITE_SPLITTER_TEST_FILE_EXCLUDE_PATTERN` | - | Optional, glob pattern to use for excluding test files or directory. </br> *It accepts pattern syntax supported by [zzglob](https://github.com/DrJosh9000/zzglob?tab=readme-ov-file#pattern-syntax) library.* |
+| `BUILDKITE_SPLITTER_TEST_FILE_PATTERN` | `spec/**/*_spec.rb` | Optional, file glob pattern for discovering test files that need to be executed. </br> *It accepts pattern syntax supported by [zzglob](https://github.com/DrJosh9000/zzglob?tab=readme-ov-file#pattern-syntax) library*. |
+| `BUILDKITE_SPLITTER_TEST_FILE_EXCLUDE_PATTERN` | - | Optional, file glob pattern to use for excluding test files or directory. </br> *It accepts pattern syntax supported by [zzglob](https://github.com/DrJosh9000/zzglob?tab=readme-ov-file#pattern-syntax) library.* |
```
Comment (TJC) [inline README.md:25]: It says `for excluding test files or directory.` -- so can I use the pattern to exclude a whole directory tree? You suggested in #52 that it had to match files.

Comment (nprizal) [reply README.md:25]: Hi @TJC, yes you can use `BUILDKITE_SPLITTER_TEST_FILE_EXCLUDE_PATTERN` to exclude a whole directory, providing you use the correct globing pattern syntax. The issue you were having in #52 seems to be caused by the issue in the identifier that you raised in #55.

Can you confirm that the exclude pattern is now working on your side?

Comment (TJC) [reply README.md:25]: I was referring to the comment in https://github.com/buildkite/test-splitter/issues/52#issuecomment-2091992853 which said "We expect the exclude pattern to take the form of a file glob, not a directory path as you're probably used to doing with Rspec!"

The description you're changing in the PR now says "file glob pattern to use for excluding test files or directory." 
That seems to imply you could pass a glob that matched a directory.

So perhaps you can see where this documentation isn't actually that clear about whether it accepts a directory, or if you actually need to be creating a glob that matches all files within a directory. Based on our discussions I think the answer is "a glob that matches all files within a directory", but to a fresh reader who comes to this project, I don't think that would be clear.

Comment (TJC) [reply README.md:25]: There's also been #58 merged since this PR was opened, which I think has changed how it works?

**Comment by wooly** [comment]
I think this has been rendered unnecessary by the change that @DrJosh9000 made to the zzglob library and subsequent 0.4.0 release of the splitter, so I'm going to close this.

**Pull request closed without merging**